### PR TITLE
Squid log webUI in readable date format #1831

### PR DESCRIPTION
**Pull request by nhirokinet**
This is an idea for #1831.

### CODE REVIEW

File: src/www/diag_logs_common.inc [hunk 1]
```diff
@@ -29,13 +29,43 @@
     POSSIBILITY OF SUCH DAMAGE.
 */
 
+require_once('guiconfig.inc');
```
Comment (fichtner) [inline src/www/diag_logs_common.inc]: this shouldn't be necessary

File: src/www/diag_logs_proxy.php [hunk 2]
```diff
@@ -7,7 +7,12 @@
 
 $logfile = "/var/log/squid/{$type}.log";
 $logclog = false;
+
 $logsplit = 2;
+if ($type === 'access' || $type === 'store') {
+    $logsplit = 1;
+    $log_datetime_format = 'unix';
```
Comment (fichtner) [inline src/www/diag_logs_proxy.php]: you should declare that variable above so that it is always declared

**Comment by nhirokinet** [comment]
I fixed them.
As I remember I added `require_once('guiconfig.inc');` to load correct timezone, but it seems just my mistake.

**Comment by fichtner** [comment]
I'm ok to try this if @AdSchellevis is also ok with it

**Comment by AdSchellevis** [comment]
@fichtner I haven't tried it out yet, but in principle I'm not agains reformatting unix timestamps, this just looks a bit too overly complicated for my taste. I also think that the default format should match the other log files (YYYY/MM/DD HH24:MI:SS if I'm not mistaken)

**Comment by nhirokinet** [comment]
@AdSchellevis This is just an opinion by just a user, I felt unixtime was better way to store here:

- In #1831 , non-default log format of squid was concern for third-party log analyzer (first reason was this).
- With lack of abstract expression of local time in PHP (and I think some other languages),  I felt unixtime is better storing format, and WebUI can immediately reformat it (and I think is nice if it's according according to users' current preferences).
- Unixtime does not lack an information about DST (but I live in a country without DST, I actually don't know how they treat DST) or timezone.
- Default log format seems to be "Apr 22 23:19:22", whose `$logsplit` should be [3](https://github.com/opnsense/core/blob/715ad3f2534c58b9ec78cdcb062ba90322539188/src/www/diag_logs_common.inc#L33). Squid cache log seems to have `YYYY/MM/DD HH24:MI:SS` format, but it [overrides](https://github.com/opnsense/core/blob/715ad3f2534c58b9ec78cdcb062ba90322539188/src/www/diag_logs_proxy.php#L10) the default.

**Comment by AdSchellevis** [comment]
@nhirokinet you're right about the format, my mistake. The code itself could be simplified, but I don't mind adding this first, since the impact is low and there's not a lot of code involved.

**Comment by fichtner** [comment]
Merged, thanks. @AdSchellevis I am not sure which simplifications you mean. Feel free to pick this up when you have time :)

**Pull request merged**